docs/didkit-packages/Node.md

```diff
@@ -43,4 +50,4 @@ const verificationMethod = DIDKit.keyToVerificationMethod('key', key);
 ## Code
 
 - [WASM](https://github.com/spruceid/didkit/tree/main/lib/web)
-- [Neon](https://github.com/spruceid/didkit/tree/main/lib/node)
+- [WASm-node](https://github.com/spruceid/didkit/tree/main/lib/node)
```

Comment:
Could there be some confusion around the Neon version and the node specific WASM package?

Reply:
I could only find 2 versions to install via npm, the WASM version, and the node specific version.  I could not find a third neon version to install.

Reply:
It's this one https://www.npmjs.com/package/@spruceid/didkit

Reply:
Thanks @sbihel.  I tried to install it via 'npm install @spruceid/didkit' and got errors.  Attached is the npm error log.  I don't have time at the moment to debug, but will look when I get a chance.  Am open to ideas/suggestions.  TIA
[2022-03-09T21_53_08_161Z-debug-0.log](https://github.com/spruceid/spruceid.dev/files/8218451/2022-03-09T21_53_08_161Z-debug-0.log)

Reply:
I see, looks like the Neon version doesn't support Windows. I think you can drop the PR for now, we'll have to decide internally what to do with the Neon version and the other packages.

Reply:
Also go errors when trying to install "@spruceid/didkit" on Windows WSL (Ubuntu).  I think I'll make one more update to Node,md to clarify what works on Linux and what works on Windows.